### PR TITLE
kompose convert --rc and kompose convert --ds not right

### PR DESCRIPTION
**Pull request by nkysg**
when i run `kompose convert --rc`
It show that 
Error: unknown flag: --rc

when i run ｀kompose convert --ds`
it shows `Error: unknown flag: --ds`

### CODE REVIEW

**Comment by pradeepto** [comment]
@nkysg Before we can do anything with your PR, you need to sign the k8s CLA. Please do so by following the link mentioned above. Thanks.

**Comment by nkysg** [comment]
I have signed CLA.

**Comment by nkysg** [comment]
I have retry @k8s-ci-robot

**Comment by procrypt** [comment]
Hey @nkysg thanks for pointing that out, but before we move forward please sign the cla and update your PR with one more additional change [here](https://github.com/kubernetes-incubator/kompose/edit/master/docs/user-guide.md) on line number 259 one more `--rc` is present please correct that too.

File: docs/user-guide.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ file "redis-rc.yaml" created
 file "web-rc.yaml" created
 ```
 
-The `*-rc.yaml` files contain the Replication Controller objects. If you want to specify replicas (default is 1), use `--replicas` flag: `$ kompose convert --rc --replicas 3`
```
Comment (procrypt) [inline docs/user-guide.md]: You need to change `--dc` to `--deployment-config` and `--rc` to `--replication-controller` also look the output too. When we run ` kompose convert --replication-controller` the output should be a file with `*-replicationcontroller.yaml` not `*-rc.yaml`.

Comment (nkysg) [inline docs/user-guide.md]: I fix this

File: docs/user-guide.md [hunk 2]
```diff
@@ -252,8 +252,8 @@ The `*-deployment.json` files contain the Deployment objects.
 $ kompose convert --replication-controller
 file "redis-svc.yaml" created
 file "web-svc.yaml" created
-file "redis-rc.yaml" created
-file "web-rc.yaml" created
+file "redis-replicationcontroller.yaml" created
+file "web-replicationcontroller.yaml" created
 ```
 
 The `*-rc.yaml` files contain the Replication Controller objects. If you want to specify replicas (default is 1), use `--replicas` flag: `$ kompose convert --replication-controller --replicas 3`
```
Comment (procrypt) [inline docs/user-guide.md]: Still there is one `*-rc.yaml` on line 259.

**Comment by procrypt** [comment]
LGTM

**Comment by nkysg** [comment]
thanks @procrypt ! this is my first commit with my poor english

**Comment by nkysg** [comment]
@procrypt I try to use git merge --squash

**Pull request closed without merging**